spinalcordtoolbox/download.py

```diff
@@ -159,7 +159,13 @@
             "https://github.com/ivadomed/multimodal-registration/releases/download/r20220512/models.zip"
         ],
         "default_location": os.path.join(__sct_dir__, "data", "deepreg_models"),
-    }
+    },
```

Comment:
Yeah, thanks!
I cloned the repo manually; thus it contained `-` instead of `_`. But the correct way should have been to use `sct_download_data`.

I reverted the https://github.com/spinalcordtoolbox/spinalcordtoolbox/commit/37447dd3dd34234ab29adea2ba4be4bfb7df9bc6 commit and added a check if `PAM50_normalized_metrics` exists in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4003/commits/274107a8df2b9c02636dded7af4a5d42b6f55b17.